books/scss/detail.scss

```diff
@@ -0,0 +1,37 @@
+.book-box {
+  .book-box-info {
+    .book-box-title {
+      h2 {
```

Comment:
Remember to use only classes

Reply:
Classes have been added to improve this point